fuzz/fuzz_targets/fuzz_int_set.rs

```diff
@@ -0,0 +1,673 @@
+#![no_main]
+//! A correctness fuzzer that checks all of the public API methods of IntSet.
+//!
+//! This fuzzer exercises the public API of IntSet and compares the results to the same operations run
+//! on a BTreeSet. Any differences in behaviour and/or set contents triggers a panic.
+//!
+//! The fuzzer input data is interpretted as a series of op codes which map to the public api methods of IntSet.
+//!
+//! Note: currently only inclusive mode IntSet's are tested.
```

Comment:
Can we file and reference an issue for all the unfuzzed work? - I immediately see both this and a todo for inverted sets

Reply:
I've got a PR coming soon with the missing functionality.